io.github.wazzaps.Fingerpaint.yml

```diff
@@ -0,0 +1,41 @@
+app-id: io.github.wazzaps.Fingerpaint
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+command: "fingerpaint"
+finish-args:
+  # GUI Permissions
+  - "--share=ipc"
+  - "--socket=wayland"
+  - "--socket=fallback-x11"
+  - "--device=dri"
+
+  # Access to udev for finding raw touchpad device
+  - "--device=all"
+  - "--filesystem=/sys:ro"
```

Comment:
this doesn't seem to be necessary.

Reply:
I need to enumerate the touchpad(s), only reasonable way to do this is with udev, which requires /sys access.

Reply:
it did find it without /sys.

also it should be noted that the use of udev that way is not guaranteed at all.